shared/middleware/helpers/storage_apps.rb

```diff
@@ -29,7 +27,7 @@ def create(value, ip_address)
   end
 
   def delete(channel_id)
-    owner, id = storage_decrypt_channel_id_checked(channel_id)
+    owner, id = storage_decrypt_channel_id(channel_id)
     raise NotFound, "channel `#{channel_id}` not found in your storage" unless owner == @storage_id
```

Comment:
I don't know... we may have some filtering in place. Either they are being filtered, or not happening in large measure right now since they are not making it high on our list. If they are showing up in HoneyBadger we will get around to doing something about them.